src/interpreter/Interpreter.as

```diff
@@ -317,7 +318,7 @@ public class Interpreter {
 	}
 
 	// Returns true if the thread needs to yield while data is requested
-	public function checkBlockingArgs(b:Block):Boolean {
+/*	public function checkBlockingArgs(b:Block):Boolean {
```

Comment:
I would recommend simply removing this function rather than commenting it. If we need to look at or retrieve the code we can always look into the history in git or on GitHub.